examples/contour/01-crds.yaml

```diff
@@ -1006,7 +1011,7 @@ spec:
                           feature gate. \n This field is immutable."
                         items:
                           x-kubernetes-map-type: atomic
-                          description: ResourceClaim references one entry in PodSpec.ResourceClaims.
+description: ResourceClaim references one entry in PodSpec.ResourceClaims.
```

Comment:
We just merged https://github.com/projectcontour/contour/pull/4972 to fix this whitespace problem for macOS, if you could rebase your branch on `main` and re-run `make generate` it should go away.